source/conf.py

```diff
@@ -41,7 +41,7 @@
 
 # General information about the project.
 project = u'CSCI 161'
-copyright = u'2018, James Hughes & Mark Daley'
+copyright = u'2021, James Hughes, Taras Mychaskiw, & Jean-Alexis Delamer'
```

Comment:
I'm definitely contributing less to this course than 102. I won't be offended if you remove me from this one.

Reply:
I suggest keepin' ya, since the editing you have done is top tier